docs/source/TaskEngineWorkflows.md

```diff
@@ -23,6 +23,10 @@ This workflow will create a server side manifest, with and/or without DRM, that
 | combine_sources   |No | This boolean indicates whether the isma/v/ts generated from the source content are to be combined into a single ismv before packaging the manifests.|true|
 | create_dref       |No | This boolean indicates whether a dref MP4 is generated for the VOD content.|true|
 | all_audio_tracks  |No | This boolean indicates whether all audio tracks are captured or only the audio tracks with the highest bitrates for each language are captured| true|
+| encrypt_ismv      |No | This boolean indicates whether the resulting ismv file should be encrypted. This is can be used to implement TransDRM	 |false|
+| playready_key     |No | The playready key used to encrypt the ismv file (if encrypt_ismv is set to true). If no playready key is provided, one will be generated through VuDRM.|""|
+| preview_thumbnails|No | Generate thumbnail assets which can then be used for video timeline previews.| false |
```

Comment:
Re-word to match the rest of the documentation. "This boolean indicates wether, etc..."